ui/src/app/utils/access-utils.tsx

```diff
@@ -186,6 +190,27 @@ export const getAccessModuleConfig = (
       AccessModule.RASLINKIDME,
       () => ({
         ...apiConfig,
+        isEnabledInEnvironment:
```

Comment:
While I was developing, I changed the text in here, so I could ensure that the logic for getInitialRTModules was working correctly.

Since the design for this component is going to be done in a separate PR, I made this component equivalent to the current Login.gov component for now, since the most recent change will result in the ID.me component being shown.